styles/modal.css

```
@@ -79,10 +98,176 @@ body.modal-open {
+  .modal-background {
+    align-items: flex-end;
+  }
+
+  .modal {
+    width: 100%;
+    max-width: 100%;
+    border-radius: 24px 24px 0 0;
+    padding: 32px 24px 48px;
+    max-height: 50vh;
+    
+    transform: translateY(100%);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)